webui/react/src/components/Table.stories.tsx

```diff
@@ -13,14 +12,14 @@ import css from './Table.module.scss';
 
 export default {
   component: Table,
-  decorators: [ RouterDecorator, ExperimentsDecorator ],
+  decorators: [ RouterDecorator ],
   title: 'Table',
 };
 
 const commandTasks: CommandTask[] = new Array(20)
   .fill(null)
   .map((_, index) => generateCommandTask(index));
-const experimentItems: ExperimentItem[] = generateExperiments(30);
+const experiments: ExperimentItem[] = generateExperiments(30);
```

Comment:
non-blocking: Not necessary for this ticket, but might be worth having storybooks for edge cases (zero experiments, 100k experiments)

Reply:
Added to our AG meet.